charts/vineyard-operator/Chart.yaml

```diff
@@ -30,10 +24,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.8.2
+version: 0.1.0
```

Comment:
I suggest follow the version of vineyard, as we are in the same repo and use the same release cycle.

Reply:
Get it.